src/GLX/glvnd_genentry.c

```diff
@@ -166,7 +179,15 @@ static void *DefaultDispatchFunc(void)
 
 static GLVNDentrypointStub GetEntrypointStub(int index)
 {
+#if defined(__CHERI_PURE_CAPABILITY__)
+    const ptraddr_t sentry_addr = __builtin_cheri_address_get(glx_entrypoint_start);
+    const ptraddr_t stub_addr = sentry_addr + (index * STUB_SIZE);
+    const void* pcc = __builtin_cheri_program_counter_get();
+    uintptr_t result_cap = (uintptr_t) __builtin_cheri_address_set(pcc, stub_addr);
+    return (GLVNDentrypointStub) __builtin_cheri_seal_entry(result_cap | 1);
```

Comment:
Why do we need | 1? If glx_entrypoint_start is a sentry then it should already have the right LSB.

Reply:
Or is it because glx_entrypoint_start is a data symbol?

Reply:
Regardless, the | 1 also is only for Morello, not all CHERI...